apis/apigee/v1alpha1/envgroupattachment_types.go

```diff
@@ -29,10 +29,10 @@ type ApigeeEnvgroupAttachmentSpec struct {
 	OrganizationRef *apigeev1beta1.OrganizationRef `json:"organizationRef"`
 
 	// Immutable. The Apigee environment group which will host the environment.
-	EnvgroupRef *EnvironmentGroupRef `json:"envgroup"`
+	EnvgroupRef *EnvironmentGroupRef `json:"envgroupRef"`
```

Comment:
qq: shall this be `envGroupRef`?  Defer to @jasonvigil

Reply:
We're not capitalizing the `g` anywhere else 😝 ; including the envgroup resource itself

Reply:
😞

Reply:
To clarify, I'm happy to take on a capitalization effort for this resource (and others) if that's the right thing to do 🔨